loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/json/LoadFlowResultDeserializer.java

```diff
@@ -68,7 +68,12 @@ public LoadFlowResult.ComponentResult deserializeComponentResult(JsonParser pars
                 }
                 case "status" -> {
                     parser.nextToken();
-                    status = LoadFlowResult.ComponentResult.Status.valueOf(parser.getValueAsString());
+                    String tempStatus = parser.getValueAsString();
+                    if (version.compareTo("1.4") < 0 && "SOLVER_FAILED".equals(tempStatus)) {
+                        // SOLVER_FAILED removed in v1.4, translated to FAILED
+                        tempStatus = LoadFlowResult.ComponentResult.Status.FAILED.name();
+                    }
+                    status = LoadFlowResult.ComponentResult.Status.valueOf(tempStatus);
```

Comment:
@jeandemanged in this case you can maybe fill the statusText with the solver failed information?